gsplat/rasterize.py

```diff
@@ -230,7 +225,8 @@ def backward(ctx, v_out_img, v_out_alpha=None):
                 v_out_img,
                 v_out_alpha,
             )
-
+        v_background = torch.matmul(v_out_img.float().view(-1, 3).t(), final_Ts.float().view(-1, 1)).squeeze()
```

Comment:
Should we have a check that (Because I believe v_background could be some overhead at rendering time)
```
v_background = None
if background. requires_grad:
    v_background = ...
```

Reply:
thanks for the advice. fixed